src/components/ExpandButton.tsx

```diff
@@ -15,6 +16,7 @@ export const ExpandButton = ({ onClick }: FoldedListboxProps) => (
       sx={{
         backgroundColor: '#F0F0F0',
         width: '100%',
+        height: `${BUTTON_HEIGHT}px`,
```

Comment:
`px` is assumed when you send in an integer directly so can use:
`height: BUTTON_HEIGHT,`